user-journey.adoc

```diff
@@ -0,0 +1,180 @@
+= User Journey for Istio/Jaeger
+
+ *Authors:* Lincoln Baxter, III and Charles Moulliard, Diógenes Rettori
+ *Version:* 0.6 - April 25, 2018
+
+This document describes the steps required to install a local Istio Openshift instance, necessary prerequisites (JDK, Maven, Launcher, Jaeger, …), and the operations to grant access for the user to the project. It also describes how the user can experiment with a mission selected using the launcher
+
+The Fabric8 Launcher provides two workflows to play with an Istio mission (zip vs. git flow), and this document will apply to either - described below:
+
+== Flows description
+
+*“Zip flow”:* Selected mission is downloaded from the web site as a zip archive and extracted to the local filesystem.
+*“Git flow”:* Selected mission is saved to a github repository/account, and an Istio application is created under the user’s local Openshift instance - within the project defined (which corresponds to the same name as the git repo created).
+
+== Prerequisites
+. *Access to a docker engine - (skip to step 2 if you already have one)*
+.. Red Hat Docker Registry - Certificate Configuration
```

Comment:
These steps are about certificates, not the docker engine, aren't they?

Also, have we confirmed which container registry will have the images?  (Also, should remove as many references to "Docker" as possible and use the generic "container" instead).

Reply:
Oh good... the lovely "don't use 'Docker'(tm)" is back ;)

Reply:
I'll review and do that..